src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -101,6 +101,9 @@ def _list(ioctx):
                 with rbd.Image(ioctx, self._image_name) as image:
                     result = image.config_list()
             else:  # pool config
+                pg_status = CephService.get_pool_pg_status(self._pool_name)
+                if 'creating+incomplete' in pg_status:
```

Comment:
Could you please add a comment why it is necessary to handle this issue here. This should help developers to understand this when they came across this code.

Reply:
Added a comment and a link to the corresponding issue it works around.